examples/asr/conf/asr_finetune/speech_to_text_hf_finetune.yaml

```diff
@@ -0,0 +1,189 @@
+name: "Speech_To_Text_HF_Finetuning"
```

Comment:
Lets add dataset name here and some comments about what this file is how to use it to train.  
or may be notebook is best, @titu1994 what do you think?

Reply:
Yep lets do that